packages/shared/models/board.ts

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose'
+
+const boardSchema = new mongoose.Schema({
+  id: String,
+  title: String,
+  description: String,
+  createdAt: Date,
+  status: String,
+  users: [
+    {
+      id: String,
+      username: String,
+      avatarUrl: String
```

Comment:
what is this field for?
The images will be stored with type BSON. But it's to heavy to store it here
https://docs.mongodb.com/manual/reference/bson-types/

Reply:
change to avatarId: Number